modules/doc/content/applications/thermal_hydraulics.md

```diff
@@ -0,0 +1,18 @@
+# Thermal Hydraulics Applications
+
+There are several MOOSE-based applications modeling thermal hydraulics, which
+are developed as part of the
+[Nuclear Energy Advanced Modeling and Simulation (NEAMS) program](https://neams.inl.gov/).
+These applications provide capabilities at various levels of fidelity vs.
+computational efficiency for a multi-layered simulation strategy. The following
+table summarizes these applications, in order of decreasing fidelity.
+
+| Application | Description | Link(s) |
```

Comment:
I would either split this table in open & not open source

also NS module should be featured?

OR add a column